question-service/package.json

```diff
@@ -15,12 +15,14 @@
     "nodemon": "^2.0.19"
   },
   "dependencies": {
-    "bluebird": "^3.7.2",
     "cors": "^2.8.5",
+    "db-migrate": "^0.11.13",
```

Comment:
Out of curiosity, what is db-migrate for?

Reply:
Oh it's actually redundant which I forgot to remove, will remove